static/css/budget.less

```diff
@@ -11,30 +17,30 @@
 
       li {
           .list_item();
-          
+
           a {
               .list_link();
               .jump-link();
           }
       }
     }
-    
-    &.financial-report a, 
+
+    &.financial-report a,
     &.performance-plan a,
     &.funding-request a span{
         &:before {
             .cf-icon();
-            content: "\e402";
+            content: "\e406";
```

Comment:
I'd have to test this (and see the associated markup) to be sure, but I think you can just use `.cf-icon-download();` in place of the explicit `content` property.